modules/go-add-infra-nodes.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="add-infra-nodes_{context}"]
+= Adding Infra Nodes
```

Comment:
```suggestion
= Adding infrastructure nodes
```
- Headings should use sentence case
- Spell out `infra` as `infrastructure` throughout this content